dev/standalone-keycloak/galaxy_ng.env

```diff
@@ -0,0 +1,33 @@
+PULP_CONTENT_PATH_PREFIX=/api/automation-hub/v3/artifacts/collections/
+
+PULP_GALAXY_API_PATH_PREFIX=/api/automation-hub/
+PULP_GALAXY_AUTHENTICATION_CLASSES=['rest_framework.authentication.SessionAuthentication', 'rest_framework.authentication.TokenAuthentication']
+PULP_GALAXY_DEPLOYMENT_MODE=standalone
+PULP_GALAXY_REQUIRE_CONTENT_APPROVAL=false
+PULP_RH_ENTITLEMENT_REQUIRED=insights
+
+PULP_ANSIBLE_API_HOSTNAME=http://localhost:5001
+PULP_ANSIBLE_CONTENT_HOSTNAME=http://localhost:24816/api/automation-hub/v3/artifacts/collections
+
+PULP_TOKEN_AUTH_DISABLED=true
+PULP_CONTENT_ORIGIN="http://localhost:24816"
+
+
+# Pulp container requires this to be set in order to provide docker registry
+# compatible token authentication.
+# https://docs.pulpproject.org/pulp_container/workflows/authentication.html
+
+PULP_TOKEN_AUTH_DISABLED=false
+PULP_TOKEN_SERVER=http://localhost:5001/token/
+PULP_TOKEN_SIGNATURE_ALGORITHM=ES256
+PULP_PUBLIC_KEY_PATH=/src/galaxy_ng/dev/common/container_auth_public_key.pem
+PULP_PRIVATE_KEY_PATH=/src/galaxy_ng/dev/common/container_auth_private_key.pem
+
+SOCIAL_AUTH_KEYCLOAK_KEY=automation-hub
```

Comment:
I am not sure if this helps, but any variable exported using `PULP_` prefix is available on `django.conf.settings` automatically.

Ex:

```bash
export PULP_SOCIAL_AUTH_KEYCLOAK_KEY=automation-hub
```

```py
from django.conf import settings
settings.get("SOCIAL_AUTH_KEYCLOAK_KEY")
```

The advantage is that you do not need to manually read `os.getenv` on settings.py to have those variables set.

Reply:
Good info.